.github/workflows/node.js.yml

```diff
@@ -31,8 +31,10 @@ jobs:
         with:
           # If no package-locks or the main package.json have changed, it should be safe to restore all node_modules as they were
           # Changing the main package.json may change the postinstall scripts, so it should be unchanged to cache install results
-          path: '**/node_modules'
-          key: ${{ runner.os }}-modules-${{ hashFiles('**/package-lock.json', 'package.json') }}
+          path: |
+            node_modules
+            packages/*/node_modules
+          key: ${{ runner.os }}-modules-${{ hashFiles('package.json', 'package-lock.json', 'packages/*/package-lock.json') }}
```

Comment:
Should include `packages/*/package.json` as well. In the case where someone accidentally updates a package.json but doesn't update the associated package-lock.json, we don't want to pull from the cache

> If dependencies in the package lock do not match those in package.json, npm ci will exit with an error, instead of updating the package lock.